src/__test__/swish.spec.ts

```diff
@@ -212,17 +214,17 @@ describe("swish", () => {
     let addDonationStub: sinon.SinonStub;
 
     beforeEach(() => {
-      getOrderByKIDStub = sinon.stub(DAO.swish, "getOrderByKID");
-      fetchStub = sinon.stub(global, "fetch");
+      getOrderByIDStub = sinon.stub(DAO.swish, "getOrderByID");
+      fetchStub = sinon.stub(fetch, "default");
```

Comment:
So the KID is not unique as I had previously thought, so `getOrderByKID` was replaced with `getOrderByID` to find the unique record.

Previously I had used the browser fetch which isn't available in the node environment, so I imported node-fetch and stubbed that instead of the global browser fetch.

Reply:
Ah, yes, it was the fetch stub I was curios about 👍